src/components/ToastNotification/ToastNotificationList.js

```diff
@@ -25,6 +30,12 @@ class ToastNotificationList extends React.Component {
     onMouseLeave && onMouseLeave();
   }
 
+  onMouseOver() {
+    this.setState({ paused: true });
+    const { onMouseOver } = this.props;
+    onMouseOver && onMouseOver();
```

Comment:
so i think these can finally simplify these to just `onMouseOver()` calls now (due to the noop). No longer need the `&&` I don't think. We added that bit in just to simplify the syntax...

Reply:
was totally an old convention that we've since updated... I will make note of this in some coding conventions wiki i'm starting next week... was trying to "lint away" as many as I could first though! But will just be trying to continue to update a code convention doc soon... ;)